### DISCUSSION TITLE
Switch to EDL Fail Failed to switch to emergency download mode

### DISCUSSION THREAD

**Discussion by zwolfinger** [pgale/lineage_15.1_Installation_TSV#16] [Q&A]
I keep getting this error when I click Download.  The ThinkSmart View is already in EDL mode at that point.  The Qualcomm driver appears to be working because QFIL sees the port.

Any idea how to progress past this issue?

![Screenshot 2024-08-09 182516](https://github.com/user-attachments/assets/61db6338-28b4-4e64-989d-0a390c2e2e5c)

**Comment by zwolfinger** [comment]
I see now that it's stuck in 900e mode.  I've ordered a deep flash cable to access 9008 mode again.  Hopefully that will be the fix.

**Reply by pgale** [reply]
I've not seen that particular error before. My guess is it's still to do with USB compatibility but that is a guess. Seeing the USB port isn't an indication of the driver working. In every case where USB issues were present in my testing and that I've seen in others testing, the USB port is always detected by the OS. It's at download time that it fails. 

I hope that your cable resolves the problem - let us know how you get on.

**Reply by zwolfinger** [reply]
The deep flash cable doesn't seem to make a difference.  Whether I boot normally, boot holding down the volume-down button, or boot holding down both volume buttons, I'm still getting only 900e mode.   I think I may have bricked the tablet.  

I've ordered a couple more tablets and hopefully will have better luck with the next one.

**Reply by zwolfinger** [reply]
Not sure what I did to the first one, but I was able to successfully install lineage  on the TSV following these instructions.  Thank you!
